packages/perseus/src/widgets/interactive-graphs/graphs/circle.tsx

```diff
@@ -37,31 +38,101 @@ function CircleGraph(props: CircleGraphProps) {
+        centerX: center[0],
+        centerY: center[1],
+    });
+    const circleRadiusPointAriaLabel = strings.circleRadiusPointAriaLabel({
+        radiusPointX: radiusPoint[0],
+        radiusPointY: radiusPoint[1],
+    });
+    const circleRadiusDescription = strings.circleRadiusDescription({
+        radius,
+    });
+    const circleOuterPointsDescription = strings.circleOuterPointsDescription({
+        point1X: center[0] + radius,
+        point1Y: center[1],
+        point2X: center[0],
+        point2Y: center[1] + radius,
+        point3X: center[0] - radius,
+        point3Y: center[1],
+        point4X: center[0],
+        point4Y: center[1] - radius,
+    });
+
     return (
-        <>
+        <g
+            // Outer circle minimal description
+            aria-label={circleGraphAriaLabel}
+            aria-describedby={`${circleId} ${radiusId} ${outerPointsId}`}
+        >
             <MovableCircle
+                id={circleId}
+                // Focusable circle aria label reads with every update
+                // because of the aria-live property in the circle <g>.
+                ariaLabel={circleShapeAriaLabel}
+                // Aria-describedby describes additional info on focus.
+                ariaDescribedBy={`${radiusId} ${outerPointsId}`}
                 center={center}
-                radius={getRadius(graphState)}
-                onMove={(c) => dispatch(actions.circle.moveCenter(c))}
+                radius={radius}
+                onMove={(c) => {
+                    setRadiusPointAriaLive("off");
+                    dispatch(actions.circle.moveCenter(c));
+                }}
             />
             <MovablePoint
+                // Radius point aria label reads with every update.
+                ariaLabel={`${circleRadiusPointAriaLabel} ${circleRadiusDescription}`}
+                // Aria-describedby describes additional info on focus.
+                ariaDescribedBy={`${outerPointsId}`}
+                // The radius point's aria-live property is set to "off" when
+                // the circle is moved, so that it doesn't override the circle's
+                // aria-live (since the point is moved along with the circle).
+                // When the radius point is moved, the aria-live is set to
+                // "polite" so that the radius is read out.
+                ariaLive={radiusPointAriaLive}
                 point={radiusPoint}
                 sequenceNumber={1}
                 cursor="ew-resize"
                 onMove={(newRadiusPoint) => {
+                    setRadiusPointAriaLive("polite");
                     dispatch(actions.circle.moveRadiusPoint(newRadiusPoint));
                 }}
             />
-        </>
+            {/* Hidden elements to provide the descriptions for the
+                circle and radius point's `aria-describedby` properties. */}
+            <g id={radiusId} style={{display: "hidden"}}>
```

Comment:
If I remember correctly, using `display: "hidden"` also hides it from the screen reader. Did you have any issues when testing this? 
Using `a11y.srOnly` might be an alternative if you do have issues 
packages/perseus/src/util/a11y.ts

Reply:
I used `display: hidden` because I don't want the screen reader (or any user) to ever actually access this element. It's only there for its text and ID so that other stuff can refer to it. It seemed to work fine in my tests, so I think this should be okay?